docs/src/InsolationExamples.md

```diff
@@ -138,10 +63,10 @@ CLIMAParameters.Planet.obliq_epoch(::EarthParameterSet) = deg2rad(97.86)
 days, lats, F5 = calc_day_lat_insolation(365, 180, param_set)
 
 title = format("g = {:.2f}, w = {:.2f}, e = {:.2f}", γ4, ϖ0, e0) # hide
-plot_day_lat_insolation(days,lats,F5,"YlOrRd", title, "insol_example6a.png")
+plot_day_lat_insolation(days,lats,F5,"YlOrRd", title, "insol_example3a.png")
 title = format("insolation diff: g' = {:.2f} - g = {:.2f}", γ4, γ0) # hide
-plot_day_lat_insolation(days, lats, F5-F0, "PRGn", title, "insol_example6b.png")
+plot_day_lat_insolation(days, lats, F5-F0, "PRGn", title, "insol_example3b.png")
 
 ```
-![](insol_example6a.png)
-![](insol_example6b.png)
\ No newline at end of file
+![](insol_example3a.png)
+![](insol_example3b.png)
```

Comment:
It would be nice to add an example showing insolation for a few days at two different locations, e.g., Pasadena (34.1478° N, 118.1445° W), e.g., in January, and Rovaniemi, Finland (66.5039° N, 25.7294° E), e.g., in June. It'll be good check that the diurnal cycle is also correct.